wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -164,6 +165,23 @@ toHttpStatus :: WalletError -> HTTP.Status
 toHttpStatus err = HTTP.Status (errHTTPCode $ toServantError err)
                                (encodeUtf8 $ describe err)
 
+transactionErrorToWalletError :: TxError.TxError -> WalletError
+transactionErrorToWalletError = \case
+    TxError.NotEnoughMoney coin ->
+        NotEnoughMoney (fromIntegral (Core.getCoin coin))
+    TxError.NotEnoughAllowedMoney coin ->
+        NotEnoughMoney (fromIntegral (Core.getCoin coin))
+    TxError.FailedToStabilize ->
+        UnknownError "Transaction process failed to stabilize."
+    TxError.OutputIsRedeem addr ->
+        OutputIsRedeem (show addr)
+    TxError.RedemptionDepleted ->
+        UnknownError "Redemption Depleted"
```

Comment:
At the cost of sounding pedagogical (sorry, not my intention) I'm wondering if we should apply the same style advocated here:

https://github.com/input-output-hk/cardano-sl/pull/2816#discussion_r183640924

Coalescing everything into a stringly-typed `UnknownError` looks like a missed opportunity here 😉

Reply:
> At the cost of sounding pedagogical

At the cost of sounding pedantic, do you mean "pedantic"? :trollface: 

This is a good point :smile: